### DISCUSSION TITLE
[@types/mjml] mjml cannot be imported without turning on esModuleInterop

### DISCUSSION THREAD

**Discussion by hfhchan-plb** [DefinitelyTyped/DefinitelyTyped#70310] [Issues with a @types package]
ts outputs the following error:

```
This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export. ts(2497)
```

when converting from `import mjml2html = require('mjml')` to `import * as mjml2html from 'mjml';` while `esModuleInterop` is disabled.

This change, however, works for moment, lodash, etc.

It looks like adding `declare namespace mjml2html {}` to `@types/mjml/index.d.ts` fixes the issue.

[a comment by an automated account is not shown]

**Comment by IanEdington** [comment]
Thanks for reporting. I haven't worked with a non esm setup. Have you confirmed that mjml works in this configuration?

**Reply by hfhchan-plb** [reply]
I can confirm it works for my CJS setup (w/ `esModuleInterop` disabled) if I manually edit the file in node_modules to have that declaration.

**Reply by IanEdington** [reply]
ok, if you put up a PR I'll approve it
